.env-dist

```diff
@@ -7,7 +7,7 @@ DJANGO_SECRET_KEY=dontusethisinprod
 # Only override this if you need something different from the default
 DJANGO_ALLOWED_HOSTS=web,localhost
 
-DATABASE_URL=postgresql://postgres@db/postgres
+DATABASE_URL=postgresql://postgres@db/buildhub2
```

Comment:
This switches the default to work on a `buildhub2` database rather than `postgres`. This is the default in settings.